EventKit/eventkit-cloud
# Adds maxWait to user activity update request to fix timeout issue

## Description of Improvement

Adds maxWait to the debounced function for user activity, so if the user is active on the page it will be updated before the warning appears.

## How to test

If you were active on the page in the create workflow there was a chance the request that updates user activity could be delayed long enough that the inactivity warning would incorrectly appear. 
To test this I set the auto logout time to be fairly low and remained active in the UI in the create workflow.

## Quality Assurance 

The following items have been updated:
 
## Screenshots

:smile:
